doc/oidc.md

```diff
@@ -28,6 +28,7 @@ xAPI resources accept tokens with the following scopes:
 | ---                    | ---                                           |
 | `all`                  | Full read/write access to all xAPI resources. |
 | `all/read`             | Read-only access to all xAPI resources.       |
+| `state`                | Read/Write access to the xAPI State Resource  |
```

Comment:
No period at the end? (I wouldn't make this comment if it weren't for the fact that all other descriptions have periods.)

Reply:
https://github.com/yetanalytics/lrsql/pull/280/commits/aab6e1a9ec15b120e3f162aa9b84ae37bb5cf23c